edpm_ansible/roles/edpm_ssh_known_hosts/tasks/main.yml

```diff
@@ -50,7 +50,7 @@
       run_once: true
       ansible.builtin.set_fact:
         ssh_known_hosts_lines: |-
-          {% for host in groups['allovercloud'] | intersect(play_hosts) %}
+          {% for host in groups['all'] | intersect(play_hosts) %}
```

Comment:
@slagle I'm wondering if it would be better if we have the dataplane-operator to set some fact and we could adapt things here according to that fact.

```
run_by: dataplane-operator

- task 
   when: run_by == "dataplane-operator"

- task 
   when: run_by != "dataplane-operator"
 ```